### ISSUE TITLE
Removed resnet50.pt is needed by driver.py

### ISSUE THREAD

**Issue by iakovos-panourgias** [CrayLabs/SmartSim-Scaling#19]
Hi,

the latest commit ( Inference scaling fix #18 ) removed the file "imagenet/resnet50.pt".

However, driver.py (line 866) is trying to add the missing file to the model.

The result is a FileNotFoundError:

```shell
    raise FileNotFoundError(f"File or Directory {file_path} not found")
FileNotFoundError: File or Directory ./imagenet/resnet50.pt not found
```

Thanks,
Iakovos

**Comment by iakovos-panourgias** [comment]
In addition, the file cpp-inference/inference_scaling_imagenet.cpp is calling "set_model_from_file" referencing "resnet50.pt"; causing an "istream reader failed" error from pytorch :

```shell

client.set_model_from_file(model_key, "./resnet50.pt", "TORCH", device_key, batch_size);

```

**Comment by al-rigazzi** [comment]
Hi @iakovos-panourgias, thanks for the heads up! We removed it because it was tied to specific hardware. While we decide how to handle this, you can create a new model by running the `model_saver.py` script contained in the `imagenet` directory if you need the GPU model, or `model_saver_cpu.py` if you need the CPU model. Let us know if this worked!

**Comment by al-rigazzi** [comment]
Hi Iakovos, yes, good catch, thanks for the issue you opened on `SmartSim`, we will get to that as soon as possible. Regarding this ticket, we will tackle it in a slightly different way:
- [ ] make `model_saver.py` and `model_saver_cpu.py` callable functions
- [ ] add a call to the model saver in the driver script, so that the correct model is generated when the study is started

**Comment by al-rigazzi** [comment]
@iakovos-panourgias we fixed the model behavior in #20, plus the fix you suggested was implemented in SmartSim's repo. I'm closing the issue now, but feel free to re-open it if something does not work for you!
